daq_macros.py

```diff
@@ -815,7 +819,7 @@ def runDozorThread(directory,
     """
     global rasterRowResultsList,processedRasterRowCount
 
-    time.sleep(0.5) #allow for file writing
+    time.sleep(1.0) #allow for file writing
```

Comment:
this is another area where the different behavior should not be propagated to other beamlines

Reply:
It will be re-tested when the MD2 rastering is being tested.